src/frontend/src/components/modals/AboutInvenTreeModal.tsx

```diff
@@ -0,0 +1,171 @@
+  }Database: ${server.database}\nDebug-Mode: ${
+    server.debug_mode ? 'True' : 'False'
+  }\nDeployed using Docker: ${
+    server.docker_mode ? 'True' : 'False'
+  }\nPlatform: ${server.platform}\nInstaller: ${server.installer}\n${
+    server.target && `Target: ${server.target}\n`
+  }Active plugins: ${JSON.stringify(server.active_plugins)}`;
+  return (
+    <Stack>
+      <Group>
+        <Text>
+          <Trans>Your InvenTree version status is</Trans>
+        </Text>
+        {data.dev ? (
+          <Badge color="blue">
+            <Trans>Development Version</Trans>
+          </Badge>
+        ) : data.up_to_date ? (
+          <Badge color="green">
+            <Trans>Up to Date</Trans>
+          </Badge>
+        ) : (
+          <Badge color="teal">
+            <Trans>Update Available</Trans>
+          </Badge>
+        )}
+      </Group>
+      <Title order={5}>
+        <Trans>Server Versions</Trans>
+      </Title>
+      <Table>
+        <tbody>
+          {fillTable(
+            [
+              {
+                ref: 'server',
+                title: <Trans>InvenTree Version</Trans>,
+                link: 'https://github.com/inventree/InvenTree/releases',
+                copy: true
+              },
+              {
+                ref: 'commit_hash',
+                title: <Trans>Commit Hash</Trans>,
+                copy: true
+              },
+              {
+                ref: 'commit_date',
+                title: <Trans>Commit Date</Trans>,
+                copy: true
+              },
+              {
+                ref: 'commit_branch',
+                title: <Trans>Commit Branch</Trans>,
+                copy: true
+              },
+              {
+                ref: 'api',
+                title: <Trans>API Version</Trans>,
+                link: `${host}api-doc/`
+              },
+              { ref: 'python', title: <Trans>Python Version</Trans> },
+              {
+                ref: 'django',
+                title: <Trans>Django Version</Trans>,
+                link: 'https://www.djangoproject.com/',
+                copy: true
+              }
+            ],
+            data.version
+          )}
+        </tbody>
+      </Table>
+      <Title order={5}>
+        <Trans>Links</Trans>
+      </Title>
+      <Table>
+        <tbody>
+          {fillTable(
+            [
+              { ref: 'doc', title: <Trans>InvenTree Documentation</Trans> },
+              { ref: 'code', title: <Trans>View Code on GitHub</Trans> },
+              { ref: 'credit', title: <Trans>Credits</Trans> },
+              { ref: 'app', title: <Trans>Mobile App</Trans> },
+              { ref: 'bug', title: <Trans>Submit Bug Report</Trans> }
+            ],
+            data.links,
+            true
+          )}
+        </tbody>
+      </Table>
+      <Group>
+        <CopyButton
+          value={copyval}
+          label={<Trans>copy version information</Trans>}
```

Comment:
Capitalize to `Copy version information`

Reply:
Done